device/dell/x86_64-dell_z9100_c2538-r0/plugins/sfputil.py

```diff
@@ -1,60 +1,329 @@
-#!/usr/bin/env python
+#!/usr/bin/python
```

Comment:
Shebang can be removed altogether, as this file is a plugin, and is not executable.

Reply:
Sure.. Thanks for the catch..